### PR TITLE
Fix placeholder to project name

### PR DESCRIPTION
**Pull request by themue**
**What this PR does / why we need it**:

Change XXX in CONTRIBUTING.md to Machine Controller.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

```release-note
none
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Origin (DCO). This document was created by the Linux Kernel community and is a
 simple statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
 
-Any copyright notices in this repo should specify the authors as "the Loodse XXX project contributors".
+Any copyright notices in this repo should specify the authors as "the Loodse Machine Controller project contributors".
```
Comment (nikhita) [inline CONTRIBUTING.md]: Either this should be changed to `the Machine Controller Authors`, or the copyright for all files will need to be updated :grimacing:

Comment (themue) [reply CONTRIBUTING.md]: Hmm, KubeOne uses the term "project contributors". IMHO the renaming of all headers should be an extra issue.

Comment (nikhita) [reply CONTRIBUTING.md]: Cool, I can follow-up on that.

**Comment by alvaroaleman** [comment]
@themue You can restart all failed tests by using `/retest`

**Comment by themue** [comment]
@alvaroaleman Thx, yeah. So far thought that `/retest` is testing all, not only the failed ones.

**Pull request merged**